compiler/gather_results.py

```diff
@@ -51,6 +51,7 @@
 SMALL_UNIX50_RESULTS = "../evaluation/results/unix50_4_1073741824/"
 BIG_UNIX50_RESULTS = "../evaluation/results/unix50_16_10737418240/"
 COARSE_UNIX50_RESULTS = "../evaluation/results/unix50-naive/"
+OASA_RESULTS = "../evaluation/buses/results/"
```

Comment:
We might want to start thinking soon about how to have per experiment scripts.

Reply:
I agree